multihost_test/bz_automation/test_automation.py

```diff
@@ -94,3 +94,21 @@ def test_cve_2010_3316(self, multihost, bkp_pam_config, compile_myxauth):
         execute_cmd(multihost, 'rm -vfr /root/.xauth')
         execute_cmd(multihost, f"userdel -rf {TUSER}")
         assert int(execute_cmd(multihost, "cat /tmp/xauthlog | wc -l" ).stdout_text.split()[0]) >= 2
+
+    def test_2082442(self, multihost, bkp_pam_config, create_localusers):
+        """
+        :title: pam_faillock prints "Consecutive login failures
+         for user root account temporarily locked" without even_deny_root
+        :id: c1802552-5151-11ed-b8c2-845cf3eff344
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=2126648
+                   https://bugzilla.redhat.com/show_bug.cgi?id=2082442
+        """
+        execute_cmd(multihost, "> /var/log/secure")
+        for file in ['system-auth', 'password-auth']:
+            execute_cmd(multihost, f"sed -i 's/pam_unix.so nullok/pam_unix.so nullok try_first_pass/g' /etc/pam.d/{file}")
```

Comment:
Why are you doing this?

Reply:
i can this try_first_pass line here in the reproducing steps in the bz , so i am putting it here in this test , i can remove it if not required

Reply:
yes this can be removed , and removed